### DISCUSSION TITLE
Add option "Skip" when uploading existing assets

### DISCUSSION THREAD

**Discussion by naboo** [craftcms/cms#10419] [Ideas]
Today we have "Replace" or "Keep both" when uploading existing assets. Would be nice to also have "Skip" :)

**Comment by brandonkelly** [comment]
That’s exactly what this “Cancel” button is for:

<img src="https://user-images.githubusercontent.com/47792/152816265-29798fef-cae0-40c4-aa2b-2e8762e5bc8b.png" width="378" height="200" alt="The dialog that displays when uploading a file with a conflicting filename, highlighting the “Cancel” button">

**Reply by naboo** [reply]
Problem is when uploading multiple assets and maybe 5 out of 10 exists. You can't click "Cancel" because it'll cancel the entire upload, no?

**Reply by rynpsc** [reply]
I think typically on OS dialogs "Skip" is used which I think would clarify the purpose of the button?
